app/Resources/translations/validators.fr.yml

```diff
@@ -170,3 +170,6 @@ legislative_candidate.district_number.invalid: "Le numéro de circonscription do
 legislative_candidate.latitude.invalid: "Cette valeur n'est pas une latitude valide."
 legislative_candidate.longitude.invalid: "Cette valeur n'est pas une longitude valide."
 legislative_candidate.carreer.invalid: "Cette carrière est invalide."
+
+legislative_campaign_contact_message.role.invalid: 'Le role que vous occupez dans la campagne est invalide.'
```

Comment:
No real need indeed. And I also removed all unneeded quoted strings in the file to remain consistent.